### PR TITLE
feat: add closeIcon customize tag close

### PR DESCRIPTION
**Pull request by xrkffgg**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Perfermance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
close https://github.com/ant-design/ant-design/issues/24848

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  Add `closeIcon` to customize Tag close icon.         |
| 🇨🇳 Chinese |新增 `closeIcon` 用以自定义 Tag 关闭按钮。           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/tag/demo/customize.md](https://github.com/ant-design/ant-design/blob/feat-tag-icon/components/tag/demo/customize.md)
[View rendered components/tag/index.en-US.md](https://github.com/ant-design/ant-design/blob/feat-tag-icon/components/tag/index.en-US.md)
[View rendered components/tag/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/feat-tag-icon/components/tag/index.zh-CN.md)

### CODE REVIEW

File: components/tag/demo/customize.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+order: 8
+title:
+  zh-CN: 自定义关闭按钮
+  en-US: Customize close
```
Comment (afc163) [inline components/tag/demo/customize.md:5]: 加个 debug

File: components/tag/index.tsx [hunk 2]
```diff
@@ -98,7 +100,16 @@ const InternalTag: React.ForwardRefRenderFunction<unknown, TagProps> = (
   };
 
   const renderCloseIcon = () => {
-    return closable ? <CloseOutlined onClick={handleCloseClick} /> : null;
+    if (closable) {
+      return closeIcon ? (
+        <div className={`${prefixCls}-customize-close`} onClick={handleCloseClick}>
```
Comment (afc163) [inline components/tag/index.tsx:105]:
```suggestion
        <div className={`${prefixCls}-close-icon`} onClick={handleCloseClick}>
```

File: components/tag/index.tsx [hunk 3]
```diff
@@ -98,7 +100,16 @@ const InternalTag: React.ForwardRefRenderFunction<unknown, TagProps> = (
...
+          {closeIcon}
+        </div>
+      ) : (
+        <CloseOutlined onClick={handleCloseClick} />
```
Comment (afc163) [inline components/tag/index.tsx:109]:
```suggestion
        <CloseOutlined className={`${prefixCls}-close-icon`} onClick={handleCloseClick} />
```

Comment (xrkffgg) [reply components/tag/index.tsx:109]: 其实默认的 有一个 `font-weight: bold;`，自定义按钮加上这个，不好看

Comment (xrkffgg) [reply components/tag/index.tsx:109]: 不过  我看视觉上。现有的去掉 font-weight 看起来也一样，那这个我直接去掉了。

**Review by afc163 — APPROVED** [review]

**Comment by zombieJ** [comment]
@xrkffgg  以后等 CI 通过后才合，我这边合 master 先修了。

截屏2020-06-11 上午10 57 45

**Comment by xrkffgg** [comment]
ok

**Comment by xrkffgg** [comment]
改了 className 后，忘更新 snap 了 😂😂

**Pull request merged**